org.eclipse.scout.rt.server.commons/pom.xml

```diff
@@ -28,6 +28,10 @@
       <groupId>org.eclipse.scout.rt</groupId>
       <artifactId>org.eclipse.scout.rt.shared</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Would require Jackson dependency in .app too. Omit and create JSON manually for this simple case?

Reply:
Removed dependency